O/OpenLSTO/bundled/patches/include.patch

```diff
@@ -0,0 +1,366 @@
+diff --git a/M2DO_LSM/include/boundary.h b/M2DO_LSM/include/boundary.h
```

Comment:
Wh... why do we need all of this? 😕

Reply:
You mean the whole patch? I have no idea... I got it to compile. Without the patch I get tons of missing definitions etc. I sort of adapted what someone did to build this from Python.

Reply:
If there's an easier fix I'd love to do it.

Reply:
How can this work at all if all these includes are necessary? 😕

Reply:
They might not be. Previously they had the `.cpp` files included into the `.h` files, but it was "Undefined" sort of errors all over the place. I'm certain there's a more succinct way, I just went along including whatever the compiler complained about.

Reply:
So this is definitely the "noob fix".